src/components/UserProfileSettings.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <v-card>
+    <v-card-title>
+      Profile settings: <b class="username"> {{ store.userName }} </b>
+    </v-card-title>
+    <v-card-text>
+      <v-container>
+        <div>
+          <h3>Username:</h3>
+          <p>{{ store.userName }}</p>
+        </div>
+        <div>
+          <h3>Email:</h3>
+          <!-- Add email from store -->
```

Comment:
Is it something you want to do before merging?

Reply:
Yes, although I don't know how to access the email currently.

Reply:
I'll just remove for now.